markdown/events/2021/bytesize-4-github-contribution-basics.md

```diff
@@ -34,3 +34,97 @@ This will cover:
+
+Personal and organisational accounts can however have many repositories within them so one is not limited to just having either one of them, it is possible to have multiple within each of these type of accounts.
+
+If you visit the nf-core organisation on github, you will see that we have 61 repositories with around 250 people, five teams, a couple of projects, some repositories - type here on find a repository and you'll see that whatever you typed in will hopefully be found. [10:03](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=603)
+
+The basics of how you can actually contribute actively on Github is following this example workflow that I found on the web, which is called fork and brunch tutorial. It is based on a tutorial that someone else actually made, so I'm sharing in a link here and all the credits actually go to that person whose efforts those are. This Github contribution basic that assumes that you have a Github account and that you're a member of the nf-core organization. Otherwise certain steps of what I'm going to display now are not working well. So this Github contribution basics basically starts with forking a repository, so imagine you want to contribute to an existing pipeline example Sarek or Eager. Then you first have to start by making a copy of that repository,  a copy which makes it possible to later on also add some pull requests. That can be done using a so-called fork, which is then basically you can go on the web page of the particular pipeline that you're interested in, and on the top right you will see that nice little fork button here. You click on it and then you have to specify where to fork this repository. So what people usually do is they don't fork it to other organization accounts that you might have access to but they actually forget to their own private accounts. [10:42](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=643)
```

Comment:
```suggestion
The basics of how you can actually contribute actively on Github is following this example workflow that I found on the web, which is called [fork and branch tutorial](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/). It is based on a tutorial that someone else actually made, so I'm sharing it in a link here and all the credits actually go to that person whose efforts those are. This Github contribution basic that assumes that you have a Github account and that you're a member of the nf-core organisation. Otherwise certain steps of what I'm going to display now are not working well. So this Github contribution basics basically starts with forking a repository, so imagine you want to contribute to an existing pipeline, for example Sarek or Eager. Then you first have to start by making a copy of that repository,  a copy which makes it possible to later on also add some pull requests. That can be done using a so-called fork, which is then basically you can go on the web page of the particular pipeline that you're interested in, and on the top right you will see that nice little fork button here. You click on it and then you have to specify where to fork this repository. So what people usually do is they don't fork it to other organisation accounts that you might have access to but they actually fork to their own private accounts. [10:42](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=643)
```